tests/src/operator/policies.rs

```diff
@@ -1,5 +1,5 @@
 #![cfg(test)]
-#![cfg(feature = "operator")]
+// #![cfg(feature = "operator")]
```

Comment:
why?

Reply:
Oops, changed it so I can get rust-analyzer to not ignore these files.

Removed my change.